### DISCUSSION TITLE
Does CC-Tweaked mean to add new api functionality compared to ComputerCraft

### DISCUSSION THREAD

**Discussion by ToxicGLaDOS** [cc-tweaked/CC-Tweaked#1548] [Q&A] [answered]
In the README.md it says
> [CC-Tweaked] continues [ComputerCraft's] legacy with improved performance and stability, along with a wealth of new features

Do those "new features" include lua api updates? In my case I'd like to see a way to query the equipped items of a turtle. In 2018 #92 asking for the same thing was posted here, but was redirected to the main ComputerCraft repo. Because of the lack of activity on [dan200/ComputerCraft](https://github.com/dan200/ComputerCraft), I assume prs won't be accepted there.

So I guess my questions are:
1. Would CC-Tweaked accept a pull request for new lua methods in general?
2. If so, would something like #92's getEquipped(side) be accepted

**Comment by SquidDev** [comment, marked as answer]
> Would CC-Tweaked accept a pull request for new lua methods in general?

So I can't guarantee that a PR will be accepted, but yes, definitely accept them :).

> If so, would something like `turtle.getEquipped(side)` be accepted

In principle, yes. There is an open issue for this (https://github.com/cc-tweaked/CC-Tweaked/issues/964) where I think there were still some questions about motivation and what the design should look like.
